docs/api/algorand/create-delegated-payment.mdx

```diff
@@ -0,0 +1,105 @@
+# CreateDelegatedPayment
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Request Parameters
+
+| Name | Type | Description |
+| ---- | ---- | ----------- | 
+| assetId         | `string` | The ID of the DeCash asset that is being sent. |
+| sender.address  | `string` | The address of the account that is sending the funds. Note that this account must have enough funds to cover both the amount that is being sent, and the [fee](#CreateDelegatedPayment). |
+| sender.amount   | `number` | The amount that is being sent. |
+| receiver        | `string` | The address of the account that is receiving the funds. Note that, if this account is not a [DelegatedWallet](#CreateDelegatedPayment), then it must be opted into the token that is being sent. | 
```

Comment:
I would link "opted into" to the concepts / token page. Let's skip it for now, and then enhance the text with relevant links when the base docs are ready.